test/api/support/ffactory.js

```diff
@@ -0,0 +1,98 @@
+const Promise = require("bluebird")
```

Comment:
We also use some of Bluebird's collection helpers that are implemented by Node's native Promises in v6. Things like `Promise.props` and `Promise.map`.